crates/store/re_data_loader/src/loader_lerobot.rs

```diff
@@ -485,6 +486,8 @@ fn load_scalar(
             format!("Failed to get field for feature {feature_key} from parquet file")
         })?;
 
+    let entity_path = field.name().to_string();
```

Comment:
I think it would be better to explicit here: do we want slashes in the field name to produce separators in the entity hierarchy?

Chose between `EntityPath::from_single_string` and `EntityPath::parse_forgiving`